code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -735,6 +775,7 @@
 /datum/tarot/reversed/justice
 	name = "VIII - Justice?"
 	desc = "May your sins come back to torment you."
+	extended_desc = "creates a random cargo crate. This can include crates cargo would otherwise not have access to at the time."
```

Comment:
```suggestion
	extended_desc = "creates a random orderable crate. This can include crates Supply would otherwise not have access to at the time."
```